src/main/java/com/stream_pi/client/window/dashboard/actiongridpane/ActionBox.java

```diff
@@ -174,6 +175,12 @@ else if(action.getActionType() == ActionType.TOGGLE)
                 }
             }
         }
+        try
+        {
+            playActionAnimation();
+        } catch (SevereException e){
+            Logger.getLogger("").warning(e.getMessage());
```

Comment:
should report the severe problem to user. use `handleSevereException`

Reply:
done